unstructured/partition/common.py

```diff
@@ -79,12 +79,12 @@ def _add_element_metadata(
     elements: List[Element] = []
     page_number: int = 1
     for layout_element in layout_elements:
-        element = normalize_layout_element(layout_element)
-        if hasattr(layout_element, "text_as_html"):
-            text_as_html: Optional[str] = layout_element.text_as_html
-        else:
+        try:
```

Comment:
Was there a reason for switching this to the `try`/`except` instead of checking if the attribute exists?

Reply:
All good! It's functionally the same here, just wanted to check if there was something that broke with `hasattr`.